### DISCUSSION TITLE
How to make taskbar always on top?

### DISCUSSION THREAD

**Discussion by totte1015** [valinet/ExplorerPatcher#3203] [Q&A]
For every game I have running and regular sized window, it overlaps with taskbar and i HATE that, every time I alt+tab out of games, the taskbar hides behind the fullscreen windowed game im playing, making it impossible to reach my shortcuts on taskbar.
How do I force taskbar to always stay on top???

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Anixx** [comment]
1. Enable Win10 taskbar in EP settings
2. Drag and drop the taskbar to the upper edge of the screen.

**Reply by totte1015** [reply]
Nonono....... I dont mean on top as in position, I mean on top as in overlaps with any game im playing or any window I have "always on top" as in priority on screen.
Why tf would I want to have it on top positioned on screen

**Comment by pyrates999** [comment]
Please respond with the details I've asked for.

**Comment by totte1015** [comment]
Windows 11 Version 23H2, 22631.3527
EP: 22621.2861.62.2
![Screenshot 2024-05-02 015825](https://github.com/valinet/ExplorerPatcher/assets/32680625/c6a247a0-dc41-47ad-9766-d7ba8f1a0559)
![Screenshot 2024-05-02 015829](https://github.com/valinet/ExplorerPatcher/assets/32680625/025cd21c-6c0a-4a29-a90d-de90b2fad5c4)
![Screenshot 2024-05-02 015833](https://github.com/valinet/ExplorerPatcher/assets/32680625/0fd93ca0-c4c5-426d-8605-9528d6e9a744)
![Screenshot 2024-05-02 015839](https://github.com/valinet/ExplorerPatcher/assets/32680625/d302e069-b68b-4970-a972-b52a3457b3df)
![Screenshot 2024-05-02 015849](https://github.com/valinet/ExplorerPatcher/assets/32680625/47d429a3-c8dc-4588-8af1-55c60858d9aa)
![Screenshot 2024-05-02 015911](https://github.com/valinet/ExplorerPatcher/assets/32680625/97c550f7-5445-4287-901c-55d12cdca362)

**Comment by pyrates999** [comment]
Please update to the current release of EP: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.2_c245b8f

**Reply by totte1015** [reply]
It did not solve my issue.

**Comment by pyrates999** [comment]
Does this happen if you uninstall EP?

**Reply by totte1015** [reply]
Yes it seems it does :/
Why the heck is it like this?

**Reply by pyrates999** [reply]
If it's happening when EP is not installed, then this is something Microsoft should be fixing.
